.github/workflows/check-pr-title.yaml

```diff
@@ -0,0 +1,51 @@
+name: Check PR title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    env:
+      PR_TITLE: ${{ github.event.pull_request.title }}
+    steps:
+      - id: release
+        name: Check for release PR format
+        if: startsWith(github.event.pull_request.title, 'Release')
+        shell: bash
+        run: |
+          if ! grep -qP '^Release [0-9]{4}-[0-9]{2}$' <<< "$PR_TITLE"; then
+            echo "::warning::Release PR title should be: 'Release <year>-<ISO week number>'"
+            exit 1
+          fi
+
+      - name: Check for Conventional Commit format
+        shell: bash
+        if: steps.release.conclusion == 'skipped'
+        run: |
+          if ! grep -qP '^\w+(\(\w+\))?:\s' <<< "$PR_TITLE"; then
+            echo "::warning::PR title should be in Conventional Commit style, e.g. 'feat: ...'"
+            exit 1
+          fi
+
+      - name: Check for conventional type allow-list
+        if: steps.release.conclusion == 'skipped'
+        shell: bash
+        run: |
+          if ! grep -qP '^(ci|db|deps|doc|env|feat|fix|fmt|merge|refactor|repo|revert|style|test|tweak)(\(\w+\))?:\s' <<< "$PR_TITLE"; then
```

Comment:
Oops, I should’ve thought to check for this in the Tamanu repo. I’d better update the Slab docs